framework/auth/core.py

```diff
@@ -147,10 +147,11 @@ def get_user(email=None, password=None, verification_key=None):
 class Auth(object):
 
     def __init__(self, user=None, api_node=None,
-                 private_key=None):
+                 private_key=None, private_link=None):
```

Comment:
I agree with @whit537 here. Unless I'm missing something, I don't think we need both `private_key` and `private_link`. @alexschiller, can you please verify?